docs/reference/modules/http.asciidoc

```diff
@@ -48,7 +48,7 @@ to `4kb`
 
 
 |`http.compression` |Support for compression when possible (with
-Accept-Encoding). Defaults to `true`.
+Accept-Encoding). Defaults to `true`. Please be aware if HTTPS is enabled, compression will be disabled (set to `false`) to mitigate potential security risks like the BREACH attack. If you want to enforce compress HTTPS traffic, please explicitly set this to `true`.
```

Comment:
```suggestion
Accept-Encoding). If HTTPS is enabled, defaults to `false`. Otherwise, defaults
to `true`.

Disabling compression for HTTPS mitigates potential security risks, such as a
https://en.wikipedia.org/wiki/BREACH[BREACH attack]. To compress HTTPS traffic,
you must explicitly set `http.compression` to `true`.
```